Core/Core/Store/Store.swift

```diff
@@ -29,7 +29,7 @@ public enum StoreChange: Equatable {
 }
 
 public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
-    public typealias EventHandler = () -> Void
+    public typealias EventHandler = (Store<U>) -> Void
```

Comment:
@vanwagonet I was having trouble with value types. see https://github.com/instructure/canvas-ios/pull/1463/files#diff-ab37ccbb42d1561e00e51f52f3eaa386R52 . Since the callback is required to initialize a store, it's hard to reference the store itself.

Reply:
In this case, since the store is already observed, can we just use the pending state from it, instead of a separate loading flag?

Reply:
In theory, but I couldn't figure out how to make it work. Combine is a mystery...